src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeImageSection.test.ts

```diff
@@ -0,0 +1,164 @@
+import { ReadyState } from '@terra-ui-packages/core-utils';
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { h } from 'react-hyperscript-helpers';
+import {
+  defaultGoogleWorkspace,
+  defaultImage,
+  defaultRImage,
+  generateTestGoogleRuntime,
+  imageDocs,
+} from 'src/analysis/_testData/testData';
+import {
+  GcpComputeImageSection,
+  GcpComputeImageSectionProps,
+} from 'src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeImageSection';
+import { ComputeImage, useComputeImages } from 'src/analysis/useComputeImages';
+import { runtimeToolLabels, terraSupportedRuntimeImageIds } from 'src/analysis/utils/tool-utils';
+import { Ajax } from 'src/libs/ajax';
+import { ComputeImageRaw } from 'src/libs/ajax/compute-image-providers/ComputeImageProvider';
+import { GetRuntimeItem } from 'src/libs/ajax/leonardo/models/runtime-models';
+import { asMockedFn } from 'src/testing/test-utils';
+
+type UseComputeImagesExport = typeof import('src/analysis/useComputeImages');
+jest.mock(
+  'src/analysis/useComputeImages',
+  (): UseComputeImagesExport => ({
+    ...jest.requireActual('src/analysis/useComputeImages'),
```

Comment:
```suggestion
    ...jest.requireActual<UseComputeImagesExport>('src/analysis/useComputeImages'),
```

Nit: best to specify a type parameter for `requireActual`. You'd think it could infer it, but...